packages/calypso-build/webpack/sass.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+const MiniCssExtractPluginWithRTL = require( 'mini-css-extract-plugin-with-rtl' );
+const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
+
+/**
+ * Return a webpack loader object containing our styling (Sass -> CSS) stack.
+ *
+ * @param {boolean}   preserveCssCustomProperties  whether Custom Properties and properties using them should be preserved in their original form
+ * @param {string[]}  includePaths                 Sass files lookup paths
+ * @param {string}    prelude                      String to prepend to each Sass file
+ *
+ * @return {Object}                                webpack loader object
+ */
+module.exports.loader = ( { preserveCssCustomProperties, includePaths, prelude } ) => ( {
+	test: /\.(sc|sa|c)ss$/,
```

Comment:
should we consider `/\.(sc|sa|c)ss$/` being passed as a parameter?

Reply:
My take: I think it's fine to leave as is, since this file is kind of explicitly in charge of style files (and kind of in a hardwired way -- all the loaders and plugins are only really relevant for style files), so I don't expect us to need to pass different extensions any time soon. And if we do, we can still turn it into a param (i.e. my beloved YAGNI paradigm at work).

Reply:
YAGNI!